src/components/BackgroundImageText/index.css

```diff
@@ -0,0 +1,34 @@
+@import "../../styles/all.css";
+
+.root {
+  position: relative;
+
+  display: inline-block;
+
+  color: $lavender;
+  -webkit-background-clip: text;
```

Comment:
You can use `background-clip`, postcss adds vendor prefixes when we need them

Reply:
If I removed the prefixes, on safari `background-clip` doesn't work, when postcss add the prefixes?

Reply:
Update autoprefixer